modules/sitemaps/sitemaps.php

```diff
@@ -149,8 +149,15 @@ private function serve_raw_and_die( $the_content_type, $the_content ) {
 		set_query_var( 'feed', 'sitemap' );
 
 		if ( '' === $the_content ) {
+			$error = __( 'No sitemap found. Please try again later.', 'jetpack' );
+			if ( current_user_can( 'manage_options' ) ) {
+				$next = date_i18n( 'F j, Y g:i a', wp_next_scheduled( 'jp_sitemap_cron_hook' ) );
```

Comment:
This date format does not seem ideal.

<img width="430" alt="screenshot 2018-10-11 at 14 53 41" src="https://user-images.githubusercontent.com/426388/46805575-b52fcd80-cd65-11e8-9a40-0c8debf2369a.png">

What if we used something like "in x hours" instead, with `human_time_diff` for example?

Reply:
Great idea. 5b02925faa45fa4c24eb99aaf2d459b82cb0ad26